test/TEST-15-DROPIN/test-dropin.sh

```diff
@@ -0,0 +1,285 @@
+#! /bin/bash
+
+set -e
+set -x
+
+_clear_service () {
+        systemctl stop $1.service 2>/dev/null || :
+        rm -f  /{etc,run,usr/lib}/systemd/system/$1.service
+        rm -fr /{etc,run,usr/lib}/systemd/system/$1.service.d
+        rm -fr /{etc,run,usr/lib}/systemd/system/$1.service.{wants,requires}
+}
+
+clear_service () {
+        _clear_service $*
+        systemctl daemon-reload
+}
+
+clear_services () {
+        for u in $*; do
+                _clear_service $u
+        done
+        systemctl daemon-reload
+}
+
+create_service () {
+        clear_service $1
+
+        cat >/etc/systemd/system/$1.service<<EOF
+[Unit]
+Description=$1 unit
+
+[Service]
+ExecStart=/usr/bin/sleep 100000
```

Comment:
There is no `/usr/bin/sleep` on Ubuntu (for now)
```
Feb 02 10:06:38 systemd-testsuite systemd[269]: a.service: Executing: /usr/bin/sleep 100000
Feb 02 10:06:38 systemd-testsuite systemd[269]: a.service: Failed at step EXEC spawning /usr/bin/sleep: No such file or directory
Feb 02 10:06:38 systemd-testsuite systemd[1]: a.service: Main process exited, code=exited, status=203/EXEC
Feb 02 10:06:38 systemd-testsuite systemd[1]: a.service: Unit entered failed state.
```

Our tests usually run something like `ExecStart=/bin/sh -c 'sleep 1000'` or `ExecStart=/bin/sleep 1000`